Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -869,22 +872,44 @@ protected virtual void CreateCollider()
                 bodyRigidbody.freezeRotation = true;
             }
 
-            if (bodyColliderContainer == null)
+            if (customBodyCollider == null)
             {
                 generateCollider = true;
-                bodyColliderContainer = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
+                customBodyCollider = CreateColliderContainer(BODY_COLLIDER_CONTAINER_NAME, playArea);
 
-                bodyCollider = bodyColliderContainer.AddComponent<CapsuleCollider>();
+                bodyCollider = customBodyCollider.AddComponent<CapsuleCollider>();
                 bodyCollider.radius = bodyRadius;
 
+                customBodyCollider.gameObject.layer = LayerMask.NameToLayer("Ignore Raycast");
+            }
+            else
+            {
+                customBodyCollider.transform.SetParent(playArea);
+                bodyCollider = customBodyCollider.GetComponent<CapsuleCollider>();
+                VRTK_PlayerObject.SetPlayerObject(customBodyCollider, VRTK_PlayerObject.ObjectTypes.Collider);
```

Comment:
This isn't being set for the auto generated container anymore and only for the injected one, I guess this line needs to move out of the if/else statement